src/PublicSite/web/js/app/atlas/LayerSelectorViewModel.js

```diff
@@ -36,5 +37,10 @@ define([
         self.selectedLayer = ko.computed(function () {
             return self.selectedRun().id ? self.selectedRun().id + "_" + self.selectedType().id : undefined;
         }, self).publishOn("active-atlas-layer");
+
+        ko.computed(function () {
+            return self.selectedType().id || "";
+        }, self).publishOn("active-atlas-type");
```

Comment:
Should there be a unit test that verifies that this event is published at the correct times?